README.md

```diff
@@ -120,8 +126,27 @@ to remove your deployment.
 
 ### experiment
 
+Manages experiment's data and configuration. See `biomage experiment --help` for more details.
+
+#### experiment pull
+
+Downloads experiment data and config files from a given environment into `BIOMAGE_DATA_PATH/experiment_id/`. See `biomage experiment pull --help` for more information, parameters and default values.
+
+E.g. to download experiment `e52b39624588791a7889e39c617f669e` data from `production`:
+
+`biomage experiment pull production e52b39624588791a7889e39c617f669e`
+
+#### experiment get
+
+List available experiments data in a given environment. See `biomage experiment get --help` for more information, parameters and default values.
+
+Example: list experiment files in `production`:
+
+`biomage experiment get production`
+
```

Comment:
If it's to list the available experiments, maybe naming it as `ls` instead of `get` would be better? It's inline with the `ls` command in Bash, and also the s3 `ls` command. It would free up `get` to be used in another command e.g. to get the metadata about an experiment.

Reply:
ok, fair enough. I actually followed kubernetes conventions to get a list of pods. I'll change it but pull already gets you all the data about an experiment so you can't `get` anything else.